### ISSUE TITLE
@JuliaRegistrator register

### ISSUE THREAD

**Issue by jdiaz97** [TidierOrg/TidierVest.jl#4]

**Comment by jdiaz97** [comment]
ok @kdpsingh I'm gonna need your help, apparently JuliaRegistrator doesn't have access to this repo and I cannot add it

**Comment by kdpsingh** [comment]
Thanks, I'll take a look at it.

**Comment by kdpsingh** [comment]
Not sure why it wasn't working for you. I checked, and all repositories in TidierOrg are registered to work with JuliaRegistrator.

I just tagged the latest commit and it looks like it worked. Some of the checks are still pending but the package is otherwise in the process of being registered.

Here's the link: https://github.com/JuliaRegistries/General/pull/90250

If the package passes all the checks, we can close this issue.
